src/sentry/middleware/health.py

```diff
@@ -19,9 +21,16 @@ def process_request(self, request):
 
         from sentry.status_checks import check_all
         from sentry.utils import json
-        problems, checks = check_all()
 
+        results = check_all()
+        problems = list(itertools.chain.from_iterable(results.values()))
         return HttpResponse(json.dumps({
```

Comment:
I don't understand this comment. By "not changing this endpoint" do you mean "not changing the return value of this endpoint" (not including severity levels, etc.?)

Also, are we at all concerned about compat?